src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/Details.cshtml

```diff
@@ -31,30 +32,61 @@
 {
     <h1>@course.Count x @course.DisplayCourseName</h1>
 
+    if (@course.FundingBandExcess != null)
+    {
+    <div class="govuk-inset-text das-inset-text--highlight">
+        <h3 class="govuk-heading-s">@course.FundingBandExcess.ApprenticesExceedingBand apprenticeships above the funding band maximum</h3>
+        <p>
+            The price for this apprenticeship is above it's
+            @if (course.FundingBandExcess.SingleFundingBandCapExceeded == null)
```

Comment:
If this logic were in the view model (i.e. add a DisplayFundingBandeExceededMessage) then it could be unit tested.

Reply:
This is here for display reasons only, I didn't want to introduce hyper link into a view model